app/models/compute_resources/foreman/model/ec2.rb

```diff
@@ -71,6 +71,18 @@ def flavors
       client.flavors
     end
 
+    def available_availability_zones
```

Comment:
To avoid making these kind of wrappers, you can write at the beginning of the file, bellow the other `delegate`:

```ruby
    delegate :security_groups, :flavors, :zones, :to => :self, :prefix => 'available'
```

available_availability_zones is a bit redundant plus we call them zones on Foreman to be more generic